src/Commands/Backup/Schedule/SetCommand.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Backup\Schedule;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+
+/**
+ * Class SetCommand
+ * @package Pantheon\Terminus\Commands\Backup\Schedule
+ */
+class SetCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+
+    /**
+     * Sets up a week-long TTL backups to be made daily and a month-long TTL to be made weekly
+     *
+     * @authorized
+     *
+     * @command backup:schedule:set
+     *
+     * @param string $site_env Site & environment to set the schedule of, in the format `site-name.env`.
+     * @option string  $day  Day of the week to make the month-long backup
+     * @option integer $hour Hour of the day to make the backups at (1-24)
+     *
+     * @usage terminus backup:schedule:set awesome-site.dev
+     *     Sets backups to occur at a random hour, with month-long TTL backup made on a random day
+     * @usage terminus backup:schedule:set awesome-site.dev --day=day_of_the_week
```

Comment:
can you disambiguate what `day_of_the_week` means? Is that an integer? 0 = monday. I guess `@usage` is different than `@example` but I think it would be good to show a command with real values somewhere.